components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -185,12 +187,13 @@ be temporary, and you can try again later.</li>
 </ul>
      ]]></string>
 
-    <!-- NavigationDelegate.ERROR_CATEGORY_URI -->
+
     <string name="error_unknown_host_title">Address Not Found</string>
+    <!-- In the example, the two URLs in markup do not need to be translated. -->
     <string name="error_unknown_host_message"><![CDATA[
 <p>The browser could not find the host server for the provided address.</p>
 <ul>
-<li>Did you make a mistake when typing the domain? (e.g. <q><strong>ww</strong>.mozilla.org</q> instead of <q><strong>www</strong>.mozilla.org</q>)</li><li>Are you certain this domain address exists?  Its registration may have expired.</li>
+<li>Did you make a mistake when typing the domain? (e.g. <q><strong>ww</strong>.mozilla.org</q> instead of <q><strong>www</strong>.mozilla.org</q>)</li><li>Are you certain this domain address exists?  It’s registration may have expired.</li>
```

Comment:
can we remove the extra space after the `?` here too (and on the following lines)?

Reply:
grammar nit: can we change `e.g.` to `e.g.,` (assuming it's not referenced in other products like this)?